### PR TITLE
Manage multiple network interfaces

### PR DESCRIPTION
**Pull request by denishamann**

### CODE REVIEW

File: src/frontend/index.ts [hunk 1]
```diff
@@ -160,9 +154,19 @@ export async function createFlowrWindow(flowrStore: Store): Promise<FlowrWindow>
...
+      const activeMacAddress = await getActiveMacAddress()
+      evt.sender.send('receiveActiveMacAddress', activeMacAddress)
+    },
+    getAllMacAddresses: async (evt: any) => {
+      const allMacAddresses = await getAllMacAddresses()
+      evt.sender.send('receiveAllMacAddresses', allMacAddresses)
+    },
+    getIpAddress: async (evt: any) => {
+      const ipAddress = await getIpAddress()
+      console.log('DENIS')
```
Comment (lmancel) [inline src/frontend/index.ts]: Remove log

File: src/frontend/index.ts [hunk 2]
```diff
@@ -160,9 +154,19 @@ export async function createFlowrWindow(flowrStore: Store): Promise<FlowrWindow>
...
+      console.log(ipAddress)
```
Comment (lmancel) [inline src/frontend/index.ts]: Remove log

**Review by lmancel — CHANGES_REQUESTED** [review]
Only some logs to remove, and the build to fix (i.e run `npm i` and commit the new package-lock.json)

**Review by hubjac1 — COMMENTED** [review]
Could you verify why the build is failing.
You should keep a listener on message "reciveMacAdress" because we have to ensure backward compatibility with Flowr 5.1 and Flowr 5.2.7

**Review by hubjac1 — CHANGES_REQUESTED** [review]
see my previous comments

File: src/frontend/index.ts [hunk 3]
```diff
@@ -160,9 +154,19 @@ export async function createFlowrWindow(flowrStore: Store): Promise<FlowrWindow>
 
       evt.sender.send('receiveConfig', config)
     },
-    getMacAddress: async (evt: any) => {
-      const usedMacAddress = await getMacAddress()
-      evt.sender.send('receiveMacAddress', usedMacAddress)
+    getActiveMacAddress: async (evt: any) => {
+      const activeMacAddress = await getActiveMacAddress()
+      evt.sender.send('receiveActiveMacAddress', activeMacAddress)
```
Comment (lmancel) [inline src/frontend/index.ts:163]: To guarantee backward compatibility with flowr, we should also keep `getMacAddress` listener

Comment (lmancel) [reply src/frontend/index.ts:163]: (it would also return the active mac address)

**Review by lmancel — CHANGES_REQUESTED** [review]

**Review by lmancel — APPROVED** [review]

**Review by lmancel — CHANGES_REQUESTED** [review]
Automatic build fails because of an out-of-sync package-lock, please run `npm i` to update it

**Review by lmancel — APPROVED** [review]

**Review by hubjac1 — APPROVED** [review]

**Pull request merged**